src/main/java/com/ericsson/eiffel/remrem/semantics/eiffelSchemas/ChangeJsonSchemaProperty.java

```diff
@@ -0,0 +1,221 @@
+	 * and other required properties to input jsons
+	 * 
+	 * @param jsonContent-
+	 *            Eiffel repo json file content sent as an input parameter to
+	 *            this method
+	 * @param jsonElementName
+	 *            - Name of the json elements as an input parameter to this
+	 *            method
+	 * @param jsonObject
+	 *            - Json object is sent as an input to this to add required json
+	 *            properties to generate event pojo's
+	 */
+	private void addAttributesToJsonSchema(JsonObject jsonContent, String jsonElementName, JsonObject jsonObject) {
+		Set<Entry<String, JsonElement>> values = jsonContent.entrySet();
+		Iterator<Entry<String, JsonElement>> keyValue = values.iterator();
+		while (keyValue.hasNext()) {
+			Entry<String, JsonElement> valueSet = keyValue.next();
+			if (valueSet.getValue().isJsonObject()) {
+				String name = valueSet.getKey();
+				oldName = jsonElementName;
+				if (name.equals(EiffelConstants.META)) {
+					isMeta = true;
+				}
+				if (name.equals(EiffelConstants.TYPE) && isMeta) {
+					isEnumType = true;
+					isMeta = false;
+				}
+				addingItemsProperties(name, valueSet.getValue(), jsonObject);
+			} else {
+				if (EiffelConstants.TYPE.equals(valueSet.getKey())) {
+					if (EiffelConstants.OBJECTTYPE.equals(valueSet.getValue().getAsString())) {
+						jsonObject.add(EiffelConstants.TYPE, valueSet.getValue());
+						if (count == 0) {
+							jsonObject.add(EiffelConstants.JAVA_TYPE,
+									parser.parse(EiffelConstants.COM_ERICSSON_EIFFEL_SEMANTICS_EVENTS
+											.concat(StringUtils.capitalize(jsonElementName)) + "\""));
+							jsonObject.add(EiffelConstants.EXTENDS_JAVA_CLASS,
+									parser.parse(EiffelConstants.COM_ERICSSON_EIFFEL_SEMANTICS_EVENTS_EVENT));
+
+						} else {
+							jsonElementName = modifyClassName(jsonElementName);
+							String newClassName = StringUtils.capitalize(jsonElementName);
+							if (jsonElementName.equals(EiffelConstants.DATA)
+									|| jsonElementName.equals(EiffelConstants.OUTCOME))
+								jsonObject.add(EiffelConstants.JAVA_TYPE,
+										parser.parse(EiffelConstants.COM_ERICSSON_EIFFEL_SEMANTICS_EVENTS
+												.concat(fileName + "" + newClassName) + "\""));
+							else
+								jsonObject.add(EiffelConstants.JAVA_TYPE, parser
+										.parse(EiffelConstants.COM_ERICSSON_EIFFEL_SEMANTICS_EVENTS.concat(newClassName)
+												+ "\""));
+						}
+						count++;
+					} else {
+						jsonObject.add(EiffelConstants.TYPE, valueSet.getValue());
+						if (jsonElementName.equals(EiffelConstants.TIME)) {
+							jsonObject.add(EiffelConstants.FORMAT, parser.parse(EiffelConstants.UTC_MILLISEC));
+						}
+					}
+				} else {
+					jsonObject.add(valueSet.getKey(), valueSet.getValue());
+					if (valueSet.getKey().equals(EiffelConstants.ENUM)) {
+						if (isEnumType) {
+							jsonObject.add(valueSet.getKey(), parser.parse(allJsonFileNames.toString()));
```

Comment:
Adding all events to the type ENUM was a simplification while this was done manually. We can not keep a single Meta Class for all events if we want to use the schemas for validating type and version. Soon a version 1.0.1 will come for an event and then the version will not be possible to validate any longer. Lets do this the correct way and have a separate Meta representation per event. Then we do not need to pass allJsonFileNames to this class.